test/test-annotation.js

```diff
@@ -193,10 +193,12 @@ describe('Annotations', function () {
         resolve(true);
       });
     });
+
+    it('should return annotations in groupA on articleA');
   });
 
   describe('AnnotationReplies', function () {
-    let PublicAnnotation, PrivateAnnotation;
+    let PublicAnnotation, PrivateAnnotation, StupidAnnotation;
 
     before(function () {
       util.addArticleAnnotation(ArticleA._id, null, 'This is a public annotation').then(newAnnotation => {
```

Comment:
Somewhat unrelated but we should probably change this util function to not save null as an entry in the list of groups, since that's not what we expect it to contain.